tests/std/tests/P0896R4_ranges_alg_find_end/test.cpp

```diff
@@ -71,9 +72,30 @@ constexpr void smoke_test() {
     }
 }
 
+constexpr void test_devcom_1559808() {
+    // Regression test for DevCom-1559808, an interaction between vector and the
+    // use of structured bindings in the constexpr evaluator.
+
+    std::vector<int> haystack(33, 42); // No particular significance to any numbers in this function

```

Comment:
@STL I believe you missed those when pushing the review comments

Reply:
@miscco Do you mean it should not use `std::`? (The test is not `using namespace std;` though.)

Reply:
Yeah, there is no reason this one test should be special in that regard. We should rather add `using namespace std;` and remove the `std::` everywhere

Reply:
According to:
```
casey@Concept:/mnt/c/STL/tests/std/tests$ grep -EL 'using[[:space:]]+namespace[[:space:]]+std' P0896R4*/test.cpp
P0896R4_P1614R2_comparisons/test.cpp
P0896R4_ranges_alg_adjacent_find/test.cpp
P0896R4_ranges_alg_equal/test.cpp
P0896R4_ranges_alg_fill/test.cpp
P0896R4_ranges_alg_fill_n/test.cpp
P0896R4_ranges_alg_find_end/test.cpp
P0896R4_ranges_alg_mismatch/test.cpp
P0896R4_ranges_iterator_machinery/test.cpp
P0896R4_ranges_to_address/test.cpp
```
there are nine P0896R4 tests that don't `using namespace std`. I will convert all of them in a different PR.